addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -91,11 +92,17 @@ def _l10n_it_edi_prepare_fatturapa_line_details(self, reverse_charge_refund=Fals
                 price_unit = line.price_unit
 
             description = line.name
-            if not is_downpayment:
-                if line.price_subtotal < 0:
-                    moves = line._get_downpayment_lines().move_id
-                    if moves:
-                        description += ', '.join([move.name for move in moves])
+
+            # Down payment lines:
+            # If there was a down payed amount that's been deducted from this move,
```

Comment:
```suggestion
            # If there was a down paid amount that has been deducted from this move,
```

Reply:
I take the shame for payed, but the `'s` was intended as an abbreviation of `has`. Maybe it's colloquial, but still.